tox.ini

```diff
@@ -9,17 +9,7 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
-
-[testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
```

Comment:
Please split the commit. "Simplifying the tox file as a result of months-in-the-past changes" is significantly different enough from "add coveralls integration" that it's the sensible thing to spread them across two commits.

Thanks!